view/GameView/HUDScreen.py

```diff
@@ -120,7 +120,7 @@ def receive_event(self, event: pygame.event.Event) -> None:
                     self.__selected_gad_prop_idx = idx
 
     def menu_button_pressed(self) -> None:
-        self.controller.to_main_menu()
+        self.parent.parent.to_settings()
```

Comment:
```suggestion
        self.parent.to_settings()
```
Bin mir aber nicht sicher